src/main/scala/Covid19/JavaWrapper.scala

```diff
@@ -2,18 +2,18 @@ package Covid19
 
 import Covid19.Protocol.Summary
 import Covid19.Sources.Jhu
+import cats.effect.{ContextShift, IO}
 import sttp.client.{HttpURLConnectionBackend, Identity, NothingT, SttpBackend}
-import sttp.client.asynchttpclient.WebSocketHandler
 
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.{ExecutionContext}
 import scala.jdk.FutureConverters._
 
 class JavaWrapper {
-  implicit val context: ExecutionContext = ExecutionContext.global
+  implicit val contextShift: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
   implicit val sttpBackend: SttpBackend[Identity, Nothing, NothingT] = HttpURLConnectionBackend()
   val source = new Jhu()
 
   def getSummaryByCountry(countryCode: String): java.util.concurrent.CompletionStage[Summary] = {
-    source.getSummaryByCountry(countryCode).map(x => x.asInstanceOf[Summary]).asJava
+    source.getSummaryByCountry(countryCode).map(x => x.asInstanceOf[Summary]).unsafeToFuture().asJava
```

Comment:
Can it be done without `asInstanceOf`?

Reply:
Do you mean don't use `unsafeToFuture()`?

Reply:
I mean `.map(x => x.asInstanceOf[Summary])`